images/macos/provision/configuration/finalize-vm.sh

```diff
@@ -20,7 +20,7 @@ if is_Monterey; then
 fi
 
 # Put documentation to $HOME root
-cp $HOME/image-generation/output/software-report/systeminfo.txt $HOME/image-generation/output/software-report/systeminfo.md $HOME/
+cp $HOME/image-generation/output/software-report/systeminfo.* $HOME/
```

Comment:
Could you please also update https://github.com/actions/runner-images/blob/main/images.CI/macos/azure-pipelines/image-generation.yml
to add one more step to print systeminfo.json file after build.

See how it is done for md file: https://github.com/actions/runner-images/blob/main/images.CI/macos/azure-pipelines/image-generation.yml#L109